views/components/ChartList.jsx

```diff
@@ -65,50 +65,64 @@ const ChartList = ({ user }) => {
           </TableRow>
         </TableHead>
         <TableBody>
-          {chartList.map((chart) => (
-            <TableRow key={chart._id}>
-              <TableCell align='center'>
-                <Link color='textPrimary' href={routes.chart(chart._id)}>
-                  {chart?.title?.toUpperCase()}
-                </Link>
-              </TableCell>
-              <TableCell align='center'>
-                {chart?.description?.toUpperCase()}
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => onDuplicateChart(chart._id)}
-                >
-                  <PlaylistAdd />
-                </Button>
-              </TableCell>
-              <TableCell align='center'>
-                <Link href={routes.editChart(chart._id)} color='textPrimary'>
-                  <Edit />
-                </Link>
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => handleShareChart(chart)}
-                >
-                  <Share />
-                </Button>
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => removeChart(chart._id)}
-                >
-                  <Delete />
-                </Button>
-              </TableCell>
-            </TableRow>
-          ))}
+          {chartList.map((chart) => {
+            const isDisabled = user.uid !== chart.owner
+            return (
+              <TableRow key={chart._id}>
+                <TableCell align='center'>
+                  <Link color='textPrimary' href={routes.chart(chart._id)}>
+                    {chart?.title?.toUpperCase()}
+                  </Link>
+                </TableCell>
+                <TableCell align='center'>
+                  {chart?.description?.toUpperCase()}
+                </TableCell>
+                <TableCell align='center'>
+                  <Button
+                    type='button'
+                    variant='text'
+                    onClick={() => onDuplicateChart(chart._id)}
+                  >
+                    <PlaylistAdd />
+                  </Button>
+                </TableCell>
+                <TableCell align='center'>
+                  <Link
+                    href={routes.editChart(chart._id)}
```

Comment:
should we change the `href` if the user can't edit the chart?
```suggestion
                    href={isEditDisabled ? '' : routes.editChart(chart._id)}
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/135/commits/3ed7958af64f50c369b6b51ea2fcd84eda50a979